components/AppLayout/AppHeader.js

```diff
@@ -1,66 +1,244 @@
+      backgroundColor: 'inherit',
+      height: 'auto',
+      width: 'auto',
+      fontSize: 20,
+      padding: '0 10px',
+    },
+  },
+  tab: {
+    padding: 8,
+    flex: '1',
+    lineHeight: `calc(${TABS_HEIGHT}px - 8px * 2)`,
+    textAlign: 'center',
+    textDecoration: 'none',
+    whiteSpace: 'nowrap',
+    fontWeight: 500,
+    letterSpacing: 0.75,
+    color: theme.palette.secondary[300],
+    [theme.breakpoints.up('md')]: {
+      color: theme.palette.secondary[500],
+      padding: '0 10px',
+    },
+  },
+  activeTab: {
+    color: theme.palette.secondary[500],
+    [theme.breakpoints.up('md')]: {
+      color: theme.palette.primary.main,
+    },
+  },
+  menuToggleButton: {
+    width: MENU_BUTTON_WIDTH,
+    height: TABS_HEIGHT,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    background: theme.palette.secondary[300],
+    color: theme.palette.common.white,
+  },
+  profileMenu: {
+    marginTop: 50,
+    backgroundColor: theme.palette.secondary.main,
+    color: theme.palette.common.white,
+    overflow: 'inherit',
+  },
+  divider: {
+    backgroundColor: theme.palette.secondary[400],
+  },
+  listIcon: {
+    color: theme.palette.secondary[300],
+    minWidth: 0,
+    paddingRight: 8,
+  },
+  login: {
+    borderRadius: 70,
+    border: `1px solid ${theme.palette.secondary[500]}`,
+  },
+}));
+
+const Links = ({ classes }) => (
+  <div className={classes.tabs}>
+    <NavLink
+      href="/articles"
+      className={classes.tab}
+      activeClassName={classes.activeTab}
+    >
+      {t`Collected Messages`}
+    </NavLink>
+    <NavLink
+      href="/replies"
+      className={classes.tab}
+      activeClassName={classes.activeTab}
+    >
+      {t`Replies`}
+    </NavLink>
+    <NavLink
+      external
+      href={EDITOR_FACEBOOK_GROUP}
+      className={cx(classes.tab, 'hidden-xs')}
+    >{t`Editor forum`}</NavLink>
+    <NavLink
+      external
+      href={PROJECT_HACKFOLDR}
+      className={classes.tab}
+    >{t`About`}</NavLink>
+  </div>
+);
 
-        @media screen and (min-width: 768px) {
-          .root {
-            padding: 0 40px;
-          }
-        }
-      `}</style>
+function AppHeader({ onMenuButtonClick, user, openLoginModal, logout }) {
+  const [anchor, setAnchor] = useState(false);
+  const classes = useStyles();
+  const theme = useTheme();
+
+  const openProfileMenu = useCallback(e => setAnchor(e.currentTarget), [
+    anchor,
+  ]);
+  const closeProfileMenu = useCallback(() => setAnchor(null), [anchor]);
+
+  return (
+    <header className={classes.root}>
+      <div className={classes.top}>
+        <div className={classes.flex}>
+          <a href="/">
+            <img
+              className={classes.logo}
+              src={mobileLogo}
+              srcSet={`${desktopLogo} ${theme.breakpoints.values.md}w`}
+              alt=""
+            />
+          </a>
+          <Box display={['none', 'none', 'inline']}>
```

Comment:
If we use fragment as `<Links>`'s root component, we can use `<Box display={['none', 'none', 'flex']}>` here to make a flex container for all items